lib/cadence/saga/concern.rb

```diff
@@ -14,9 +14,21 @@ def run_saga(&block)
         logger.error("Saga execution aborted: #{error.inspect}")
         logger.debug(error.backtrace.join("\n"))
 
-        saga.compensate
+        if compensable?(error, configuration)
```

Comment:
nit: you need to explicitly deconstruct the hash here with `**` otherwise Ruby 2.7 is going to complain